proxy/Main.cc

```diff
@@ -1903,7 +1912,12 @@ main(int /* argc ATS_UNUSED */, const char **argv)
       if (delay_p && ink_atomic_cas(&delay_listen_for_cache_p, 0, 1)) {
         Debug("http_listen", "Delaying listen, waiting for cache initialization");
       } else {
-        start_HttpProxyServer(); // PORTS_READY_HOOK called from in here
+        if (num_accept_threads == 0) {
+          accept_per_thread_cont = new AcceptPerThreadCont;
+          eventProcessor.schedule_in(accept_per_thread_cont, HRTIME_MSECONDS(50), ET_CALL);
```

Comment:
This just picks a random thread to have the accept in? Is that what it did before?

Reply:
before it is just started on the main thread without checking if the et_net threads are initialized.